service/src/main/kotlin/fi/espoo/evaka/attendance/MobileUnitController.kt

```diff
@@ -180,20 +196,20 @@ WITH present_children AS (
     ) p
     GROUP BY p.unit_id
 ), present_staff AS (
-    SELECT g.daycare_id AS unit_id, sum(sa.count) AS count
+    SELECT g.daycare_id AS unit_id, sum(sa.capacity) AS capacity, sum(sa.count) AS count
     FROM daycare_group g
     JOIN (
-        SELECT sa.group_id, sa.count + sa.count_other AS count FROM staff_attendance sa
+        SELECT sa.group_id, 7 as capacity, sa.count + sa.count_other AS count FROM staff_attendance sa
```

Comment:
Shouldn't `capacity` be something like `7 * (sa.count + sa.count_other)`?

Reply:
Great catch, thanks!

While testing I decided to add handling to return Infinity utilization if there are kids but no staff (should not happen, but might if someone forgets to add the staff).